calour/ratio_experiment.py

```diff
@@ -0,0 +1,254 @@
+'''
+ratio experiment (:mod:`calour.ratio_experiment`)
+=======================================================
+
+.. currentmodule:: calour.ratio_experiment
+
+Classes
+^^^^^^^
+.. autosummary::
+   :toctree: generated
+
+   RatioExperiment
+'''
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--,  Calour development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from logging import getLogger
+
+import numpy as np
+import pandas as pd
+import scipy.stats
+from statsmodels.stats.multitest import multipletests
+
+from .experiment import Experiment
+from .util import _to_list
+from .analysis import _new_experiment_from_pvals, _CALOUR_DIRECTION, _CALOUR_STAT
+
+logger = getLogger(__name__)
+
+
+class RatioExperiment(Experiment):
+    '''This class stores log-ratio data and corresponding analysis methods
+
+    This is a child class of :class:`.Experiment`.
+
+    Parameters
+    ----------
+    data : numpy.ndarray or scipy.sparse.csr_matrix
+        The abundance table for OTUs or ASVs. Samples
```

Comment:
this is ratio? pls correct the `Attributes` doc too

Reply:
fixed